src/it/test-multi-module-parent/verify.groovy

```diff
@@ -2,6 +2,14 @@ import org.codehaus.plexus.util.FileUtils;
 
 String log = FileUtils.fileRead(new File(basedir, "build.log"));
 
+if (!log.contains("snyk executable path: " + System.getenv("SNYK_CLI_EXECUTABLE"))) {
```

Comment:
I suggest moving these assertions to a single scenario to avoid noise in what we're testing.  e.g. Version logging and Path logging happens every time so they can go in the basic "no dependencies" test and nothing else since it'll be covered there.